### DISCUSSION TITLE
go-cqhttp #1795那个mac qq的问题

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#386] [一般讨论]
https://github.com/Mrs4s/go-cqhttp/issues/1795
这个还是得等go-cq那边更新才有解吗，虽然用rc4或者他们dev分支里最新的也一样是那种链接就是了（

**Comment by Tsuk1ko** [comment]
等 gocq 修，或者祈祷 @MisakaTAT 会修🥹

**Comment by MisakaTAT** [comment]
我这边暂时无法复现，能否提供一下准确的复现方法？是群聊有问题还是群聊与私聊均有问题，所使用的NT QQ版本是？

**Reply by A2Sumie** [reply]
QQ是MAC QQ(NT) 6.9.2 (7809,6244) 
方法 搜图必报错，图片是NT QQ发的就能触发，gocq里日志里的就是https://github.com/Mrs4s/go-cqhttp/issues/1795 那种格式，和手机发出来的不一样
群聊私聊都不行

**Reply by MisakaTAT** [reply]
我用的是 6.8.8 (6943)，我更新下再试试

**Reply by MisakaTAT** [reply]
我在 `MAC QQ NT 6.9.2（7809）` 下测试了 `iPad` 和 `Android Watch` 协议，群聊与私聊均未复现该问题，无论是复制图片还是截图，收到的图片均可正常访问。

**Reply by MisakaTAT** [reply]
<img width="1325" alt="image" src="https://user-images.githubusercontent.com/32693466/212000058-d47d2001-af7d-4c25-8c7a-c9001630e71e.png">

**Reply by MisakaTAT** [reply]
https://github.com/CodeLaboratory/go-cqhttp/releases/tag/v2.0.0 要不你试试这个，只需要替换 go-cqhttp 主程序然后执行即可，顺便问一下你用的什么协议？

**Reply by A2Sumie** [reply]
> https://github.com/CodeLaboratory/go-cqhttp/releases/tag/v2.0.0 要不你试试这个，只需要替换 go-cqhttp 主程序然后执行即可，顺便问一下你用的什么协议？

IPAD协议，之前AndroidPhone协议也是1795里那种搜图也会失败，原版RC4和前两天最新的dev都试过，我试一下您的版本

**Reply by A2Sumie** [reply]
> <img alt="image" width="1325" src="https://user-images.githubusercontent.com/32693466/212000058-d47d2001-af7d-4c25-8c7a-c9001630e71e.png">


<img width="964" alt="image" src="https://user-images.githubusercontent.com/89308692/212010938-653a221a-be3e-465f-80fd-2759a69333cb.png">
您这个也不行，我重新在mac上登录一次看看，神奇

我动的其他bot的，我改一下这个bot对接的看看能不能正常搜图 并不行，我去重新登陆了

**Reply by A2Sumie** [reply]
> https://github.com/CodeLaboratory/go-cqhttp/releases/tag/v2.0.0 要不你试试这个，只需要替换 go-cqhttp 主程序然后执行即可，顺便问一下你用的什么协议？


重登，账密换扫码不行。换号也不行。把QQ改成以Rosetta运行也不行。都是1795那样的链接并且这边的搜图bot读取不出。
Mac上没装梯子，翻回国内这个还没有试。

**Reply by MisakaTAT** [reply]
有点奇怪，我这边怎么测试都复现不出来 🥹

**Reply by MisakaTAT** [reply]
包括群里的机器人测试也是正常的
<img width="751" alt="image" src="https://user-images.githubusercontent.com/32693466/212018169-a84edd40-16bd-4d25-a37f-61c97fbb07da.png">

**Reply by A2Sumie** [reply]
<img width="1230" alt="image" src="https://user-images.githubusercontent.com/89308692/212021732-f638b114-6bc8-4df0-83a5-c09792f136e6.png">
血压高了
忽悠国内朋友升NT版帮我试试，这NT版不走代理也挺麻的
（版本今天刚被推上去，应该没什么影响，大概）

**Reply by MisakaTAT** [reply]
我刚刚又发现有更新，但测试下来依旧正常（
<img width="480" alt="image" src="https://user-images.githubusercontent.com/32693466/212022737-fd5dc63f-cf85-4aa4-9673-7f39c3a33f9f.png">
